CMakeLists.txt

```diff
@@ -0,0 +1,308 @@
+cmake_minimum_required(VERSION 2.6)
+
+project("openscap")
+set(OPENSCAP_VERSION_MAJOR "1")
+set(OPENSCAP_VERSION_MINOR "3")
+set(OPENSCAP_VERSION_PATCH "0")
+set(OPENSCAP_VERSION "${OPENSCAP_VERSION_MAJOR}.${OPENSCAP_VERSION_MINOR}.${OPENSCAP_VERSION_PATCH}")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/")
+
+if(CMAKE_BUILD_TYPE STREQUAL "Release" OR CMAKE_BUILD_TYPE STREQUAL "RelWithDebInfo")
+	set(NDEBUG 1)
+endif()
+
+# ---------- INCLUDES CMAKE MODULES
+
+include(GNUInstallDirs)
+include(FindPkgConfig)
+include(CheckLibraryExists)
+include(CheckFunctionExists)
+include(CheckIncludeFile)
+include(CheckCSourceCompiles)
+include(CPack)
+
+# ---------- DEPENDENCIES
+
+# ACL
+find_package(ACL)
+if(ACL_FOUND)
+	CHECK_LIBRARY_EXISTS(acl acl_extended_file "" HAVE_ACL_EXTENDED_FILE)
+	CHECK_INCLUDE_FILE(acl/libacl.h HAVE_ACL_LIBACL_H)
+	CHECK_INCLUDE_FILE(sys/acl.h HAVE_SYS_ACL_H)
+endif()
+
+# APTPKG
+find_package(AptPkg)
+
+# BLKID
+find_package(Blkid)
+if(BLKID_FOUND)
+	CHECK_LIBRARY_EXISTS(blkid blkid_get_tag_value "" HAVE_BLKID_GET_TAG_VALUE)
+endif()
+
+# CAP
+find_package(Cap)
+if(CAP_FOUND)
+	CHECK_LIBRARY_EXISTS(cap cap_get_pid "" HAVE_CAP_GET_PID)
+	CHECK_INCLUDE_FILE(sys/capability.h HAVE_SYS_CAPABILITY_H)
+endif()
+
+# CURL
+find_package(CURL)
+
+# DBUS
+find_package(DBus)
+
+# DOXYGEN
+find_package(Doxygen)
+
+# GCONF2
+find_package(GConf)
+
+# LDAP
+find_package(Ldap)
+
+# OpenDbx
+find_package(OpenDbx)
+
+# PCRE
+find_package(PCRE REQUIRED)
+
+# PERL
+find_package(PerlLibs)
+
+# POPT
+find_package(Popt)
+
+# PROCPS
+find_package(Procps)
+if(PROCPS_FOUND)
+	CHECK_LIBRARY_EXISTS(procps dev_to_tty "" HAVE_DEV_TO_TTY)
+	CHECK_INCLUDE_FILE(proc/devname.h HAVE_PROC_DEVNAME_H)
+endif()
+
+# PYTHON
+find_package(PythonLibs 2 EXACT)
+find_package(PythonInterp)
+
+# RPM
+find_package(RPM)
+if(RPM_FOUND)
+	CHECK_LIBRARY_EXISTS(rpm rpmReadConfigFiles "" HAVE_RPMREADCONFIGFILES)
+	CHECK_LIBRARY_EXISTS(rpm headerFormat "" HAVE_HEADERFORMAT)
+	CHECK_LIBRARY_EXISTS(rpmio rpmFreeCrypto "" HAVE_RPMFREECRYPTO)
+	CHECK_LIBRARY_EXISTS(rpm rpmFreeFilesystems "" HAVE_RPMFREEFILESYSTEMS)
+	set(HAVE_RPMVERCMP 1)
+endif()
+if (RPM46_FOUND)
+	set(HAVE_RPM46 1)
+endif()
+if (RPM47_FOUND)
+	set(HAVE_RPM47 1)
```

Comment:
IMO just change teh references instead of having 2 equivalent variables.

Reply:
OK, great idea, thanks